src/components/Blocking/ReportPostAnonymous.js

```diff
@@ -137,7 +137,7 @@ const styles = StyleSheet.create({
     borderTopLeftRadius: 20
   },
   draggableIcon: {
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The update of the background color from `COLORS.lightgrey` to `COLORS.gray100` should be fine if it matches the new design specifications. However, make sure that this color change doesn't negatively impact the user interface's visibility or user experience.
```diff
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->